process.c

```diff
@@ -4349,8 +4349,8 @@ rb_call_proc__fork(void)
  *     Process._fork   -> integer
  *
  *  An internal API for fork. Do not call this method directly.
- *  Currently, this is called via +Kernel.#fork+, +Process.fork+, and
- *  +popen+ with +"-"+.
+ *  Currently, this is called via <tt>Kernel.#fork</tt>, +Process.fork+, and
+ *  +popen+ with <tt>"-"</tt>.
```

Comment:
I think the markups are extraneous, and no global `popen` method.
```suggestion
 *  Currently, this is called via Kernel#fork, Process.fork, and
 *  IO.popen with <tt>"-"</tt>.
```